README.md

```diff
@@ -10,12 +10,14 @@ The Stripe package has the following features:
 - Shortcuts to access the Stripe API
 - Support for webhooks
 
-In most cases, you will be using the provided shortcuts to access the API. For example, you could use the API directly by doing an HTTP request like this:
+In most cases, you will use the provided shortcuts to access the API. For example, you could use the API directly by making an HTTP request like this:
 
 ```js
 let res = pkg.stripe.api.get('/v1/customers');
 ```
 
+For further details about the Stripe API, please take a look at the [documentation.](https://docs.stripe.com/api)
+
 # Configuration
```

Comment:
Let's use the same format for all configuration parameters, check the mandrill's readme format: https://github.com/slingr-stack/mandrill-package/?tab=readme-ov-file#api-key